mirage/factories/file.ts

```diff
@@ -0,0 +1,62 @@
+import { Factory, faker } from 'ember-cli-mirage';
+import config from 'ember-get-config';
+
+import { guid } from './utils';
+
+const { OSF: { apiUrl } } = config;
+
+export default Factory.extend({
+    id(i: number) {
+        return guid(i, 'file');
+    },
+    name() {
+        return faker.system.commonFileName(faker.system.commonFileExt(), faker.system.commonFileType());
+    },
+    extra: {
+        hashes: {
+            md5: `${faker.random.uuid().replace(/-/g, '')}`,
+            sha256: `${faker.random.uuid().replace(/-/g, '')}`,
+        },
+        downloads: faker.random.number(1000),
+    },
+    kind: 'file',
+    last_touched() {
+        return faker.date.past(5);
+    },
+    materialized_path: `/${faker.system.commonFileName(faker.system.commonFileExt(), faker.system.commonFileType())}`,
+    date_modified() {
+        return faker.date.recent();
+    },
+    date_created() {
+        return faker.date.past(5);
+    },
+    guid(i: number) {
+        return guid(i, 'file');
+    },
+    currentVersion() {
+        return faker.random.number(200);
+    },
+    provider: 'osfstorage',
+    path(i: number) {
+        return `/${i}`;
+    },
+    currentUserCanComment: true,
+    checkout: null,
+    tags() {
+        return faker.lorem.words(5).split(' ');
+    },
+    size() {
+        return faker.random.number(1000000000);
+    },
+    normalLinks(i: number) {
```

Comment:
Minor: I might prefer moving these to the mirage serializer. It can be confusing when we're sort of lying to one part of the application to get data to another part (telling mirage this is an attribute so the serializer can move it elsewhere).